src/main/java/eu/yals/telegram/TelegramObject.java

```diff
@@ -10,44 +11,33 @@
 import static eu.yals.constants.App.NO_VALUE;
 
 /**
- * Contains message, username and other useful info from {@link org.telegram.telegrambots.meta.api.objects.Update}
+ * Contains message, username and other useful info from {@link org.telegram.telegrambots.meta.api.objects.Update}.
  *
  * @since 2.4
  */
-public class TelegramObject {
+@Data
+public final class TelegramObject {
 
     private TelegramArguments arguments = TelegramArguments.EMPTY_ARGS;
     private TelegramCommand command = TelegramCommand.UNKNOWN;
     private String username = NO_VALUE;
 
     private String userMessage;
 
-    private TelegramObject(Update update) {
+    private TelegramObject(final Update update) {
         this.parseUpdate(update);
         this.parseUserMessage();
     }
 
-    static TelegramObject createFromUpdate(Update update) {
+    static TelegramObject createFromUpdate(final Update update) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5409395)